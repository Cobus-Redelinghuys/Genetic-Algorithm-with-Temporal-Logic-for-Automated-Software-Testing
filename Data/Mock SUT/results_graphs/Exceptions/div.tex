Table \ref{tab:HP:GA:Exceptions:diversity in populations} tabulates the final average, standard deviations, and overall average for the diversity in populations that the different GATLAST configurations configured with different exceptions weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
exceptions & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.51 & 0.086 & 0.742\\\hline
0.25 & 0.562 & 0.098 & 0.746\\\hline
0.5 & 0.602 & 0.057 & 0.752\\\hline
0.75 & 0.532 & 0.058 & 0.745\\\hline
1.0 & 0.57 & 0.116 & 0.755\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the diversity in populations metric that the of different exceptions configured GATLASTs were achieved}
\label{tab:HP:GA:Exceptions:diversity in populations}
\end{table}