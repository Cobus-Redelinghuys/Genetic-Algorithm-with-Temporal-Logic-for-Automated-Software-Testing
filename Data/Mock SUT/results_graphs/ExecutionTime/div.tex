Table \ref{tab:HP:GA:ExecutionTime:diversity in populations} tabulates the final average, standard deviations, and overall average for the diversity in populations that the different GATLAST configurations configured with different execution time weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
execution time & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.558 & 0.062 & 0.744\\\hline
0.25 & 0.58 & 0.081 & 0.765\\\hline
0.5 & 0.602 & 0.057 & 0.752\\\hline
0.75 & 0.554 & 0.133 & 0.751\\\hline
1.0 & 0.606 & 0.094 & 0.767\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the diversity in populations metric that the of different execution time configured GATLASTs were achieved}
\label{tab:HP:GA:ExecutionTime:diversity in populations}
\end{table}