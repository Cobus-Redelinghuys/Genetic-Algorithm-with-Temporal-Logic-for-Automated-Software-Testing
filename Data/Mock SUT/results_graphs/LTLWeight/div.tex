Table \ref{tab:HP:GA:LTLWeight:diversity in populations} tabulates the final average, standard deviations, and overall average for the diversity in populations that the different GATLAST configurations configured with different ω₁ weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
ω₁ & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.602 & 0.053 & 0.755\\\hline
0.25 & 0.482 & 0.096 & 0.722\\\hline
0.5 & 0.602 & 0.057 & 0.752\\\hline
0.75 & 0.56 & 0.049 & 0.755\\\hline
1.0 & 0.528 & 0.115 & 0.74\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the diversity in populations metric that the of different ω₁ configured GATLASTs were achieved}
\label{tab:HP:GA:LTLWeight:diversity in populations}
\end{table}