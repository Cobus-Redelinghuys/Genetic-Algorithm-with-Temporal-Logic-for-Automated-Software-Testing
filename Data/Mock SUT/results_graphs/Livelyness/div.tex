Table \ref{tab:HP:GA:Livelyness:diversity in populations} tabulates the final average, standard deviations, and overall average for the diversity in populations that the different GATLAST configurations configured with different proper termination weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
proper termination & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.592 & 0.038 & 0.786\\\hline
0.25 & 0.52 & 0.097 & 0.726\\\hline
0.5 & 0.602 & 0.057 & 0.752\\\hline
0.75 & 0.6 & 0.101 & 0.76\\\hline
1.0 & 0.56 & 0.075 & 0.749\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the diversity in populations metric that the of different proper termination configured GATLASTs were achieved}
\label{tab:HP:GA:Livelyness:diversity in populations}
\end{table}