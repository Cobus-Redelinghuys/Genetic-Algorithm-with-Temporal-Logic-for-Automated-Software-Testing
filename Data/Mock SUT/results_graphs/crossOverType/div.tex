Table \ref{tab:HP:GA:crossOverType:diversity in populations} tabulates the final average, standard deviations, and overall average for the diversity in populations that the different GATLAST configurations configured with different crossover type weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
crossover type & Final average & Final standard deviation & Overall average\\
\hline
N-point crossover & 0.64 & 0.082 & 0.784\\\hline
One point crossover & 0.602 & 0.057 & 0.752\\\hline
Segmented crossover & 0.524 & 0.109 & 0.727\\\hline
Uniform crossover & 0.64 & 0.04 & 0.812\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the diversity in populations metric that the of different crossover type configured GATLASTs were achieved}
\label{tab:HP:GA:crossOverType:diversity in populations}
\end{table}