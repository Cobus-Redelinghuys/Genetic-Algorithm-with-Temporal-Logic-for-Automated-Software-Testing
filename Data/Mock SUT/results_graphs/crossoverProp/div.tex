Table \ref{tab:HP:GA:crossoverProp:diversity in populations} tabulates the final average, standard deviations, and overall average for the diversity in populations that the different GATLAST configurations configured with different crossover probability weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
crossover probability & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.408 & 0.04 & 0.703\\\hline
0.25 & 0.48 & 0.049 & 0.683\\\hline
0.5 & 0.602 & 0.057 & 0.752\\\hline
0.75 & 0.648 & 0.081 & 0.802\\\hline
1.0 & 0.552 & 0.052 & 0.781\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the diversity in populations metric that the of different crossover probability configured GATLASTs were achieved}
\label{tab:HP:GA:crossoverProp:diversity in populations}
\end{table}