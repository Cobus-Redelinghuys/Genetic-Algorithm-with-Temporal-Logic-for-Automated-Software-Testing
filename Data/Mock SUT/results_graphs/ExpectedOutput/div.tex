Table \ref{tab:HP:GA:ExpectedOutput:diversity in populations} tabulates the final average, standard deviations, and overall average for the diversity in populations that the different GATLAST configurations configured with different expected output weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
expected output & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.618 & 0.085 & 0.769\\\hline
0.25 & 0.582 & 0.103 & 0.766\\\hline
0.5 & 0.602 & 0.057 & 0.752\\\hline
0.75 & 0.466 & 0.096 & 0.711\\\hline
1.0 & 0.516 & 0.121 & 0.728\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the diversity in populations metric that the of different expected output configured GATLASTs were achieved}
\label{tab:HP:GA:ExpectedOutput:diversity in populations}
\end{table}